### ISSUE TITLE
Support `docker` managers

### ISSUE THREAD

**Issue by lebaudantoine** [openfun/renovate-configuration#5]
## Purpose

Renovate supports updating Docker Compose and Dockerfile dependencies.

## Proposal

- [ ] Enable `dockerfile` and `docker-composer` packages managers in the `default.json` configuration.
- [ ] Create a new package rule for the docker managers' category in the `default.json` configuration.

WDYT @jbpenrath?

**Comment by jbpenrath** [comment]
I saw this option but as there is no process at fun to upgrade docker images, I didn't implement it.

I think we have to poll @openfun/core team to know their interest to enable this option. From my point of view, I'm afraid this kind of update will be very time consuming or we will pin too much images.

Maybe a good workaround would be to use a larger update timeframe than for others. (Each two months instead of once a week ?).

**Comment by lebaudantoine** [comment]
Great! Let's conduct a poll to gather opinions. Do you want me to open a discussion in the tech slack chan?
I've created an issue, to follow a discussion during one of our data stand-up meetings. Considering a larger update timeframe could be a nice tradeoff!

**Comment by jbpenrath** [comment]
Yup good idea ! I let you open the discussion on the tech slack chan
